doc/index.rst

```diff
@@ -48,26 +66,23 @@ IP-XACT Resources
   * `IP-XACT <http://accellera.org/downloads/standards/ip-xact>`_ at `Accellera <http://accellera.org>`_
 
 
-
 Contributors
 ************
 
 * `Patrick Lehmann <https://github.com/Paebbels>`_ (Maintainer)
-
+* `Unai Martinez-Corral <https://GitHub.com/umarcor/>`__
+* `and more... <https://GitHub.com/edaa-org/pyEDAA.IPXACT/graphs/contributors>`__
 
 
 License
 *******
 
-This library is licensed under **Apache License 2.0**.
+This Python library (source code) is licensed under `Apache License 2.0 <License>`.
 
-------------------------------------
-
-.. |docdate| date:: %b %d, %Y - %H:%M
```

Comment:
Don't we want to keep the date?

Reply:
It's shown already because `html_last_updated_fmt = "%d.%m.%Y"` is added to `doc/conf.py`. See the footer of https://edaa-org.github.io/pyEDAA.IPXACT